src/rustdoc-json-types/lib.rs

```diff
@@ -115,6 +115,35 @@ pub enum Visibility {
     },
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub struct DynTrait {
+    /// All the traits implemented. The first gives the vtable, and the rest must be auto traits.
```

Comment:
The order doesn't matter.

```suggestion
    /// All the traits implemented. One of them gives the vtable, and the rest must be auto traits.
```

Reply:
TIL, thanks